rotkehlchen/chain/ethereum/modules/curve_lend/decoder.py

```diff
@@ -0,0 +1,27 @@
+from typing import TYPE_CHECKING, Final
+
+from rotkehlchen.chain.evm.decoding.curve_lend.decoder import CurveLendCommonDecoder
+from rotkehlchen.chain.evm.types import string_to_evm_address
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.ethereum.node_inquirer import EthereumInquirer
+    from rotkehlchen.chain.evm.decoding.base import BaseDecoderTools
+    from rotkehlchen.user_messages import MessagesAggregator
+
+LLAMALEND_LEVERAGE_ZAP: Final = string_to_evm_address('0x3294514B78Df4Bb90132567fcf8E5e99f390B687')
```

Comment:
Is this used anywhere else? If not then just hard code it as a constructor argument. No point creating a temporary variable for nothing.